src/targets/github.ts

```diff
@@ -219,6 +220,25 @@ export class GithubTarget extends BaseTarget {
     }
   }
 
+  public async getRevisionChanges(
+    version: string,
+    revision: string
+  ): Promise<Changeset> {
+    const changelog = await getFile(
+      this.github,
+      this.githubRepo.owner,
+      this.githubRepo.repo,
+      this.githubConfig.changelog,
+      revision
+    );
+    const changes = (changelog && findChangeset(changelog, version)) || {
```

Comment:
You've just identified a bug which I was gonna address some other part, thanks a lot! Will add you to the follow up.

Reply:
👍🏻

Reply:
Ah, just checked and this change actually fixes the issue you've spotted. We now call `getRevisionChanges` with the raw version string instead of the constructed tag name so we are good.

Very good catch tho! 👏🏻